src/network/gamehost.h

```diff
@@ -41,6 +41,9 @@ struct Client;
  * launch, as well as dealing with the actual network protocol.
  */
 struct GameHost : public GameController {
+	/** playernumber 0 identifies the spectators */
```

Comment:
I asked @gunchleoc about this and she told be its the specatator, but maybe the magic '0' user / player is used with overloaded semantics?

Reply:
I haven't checked, but it could indeed well be that player 0 is a spectator in some parts of the codes. That doesn't seem to be the case for the settings.user structure, though.

Reply:
Player 0 is indeed the spectator for the player slots. This variable is used in client slots though, so it doesn't make sense there.

Reply:
No I am finally confused about player slots, client slots, etc and have no idea what to do.

Reply:
Found it!

```
int16_t GameHost::get_local_playerposition() {
	return d->settings.users.at(0).position;
}
```

So, client `0` is the host, and player `0` is the spectator. We're dealing with the host here.